crates/core_simd/src/vector/float.rs

```diff
@@ -35,6 +35,13 @@ macro_rules! impl_float_vector {
             pub fn abs(self) -> Self {
                 unsafe { crate::intrinsics::simd_fabs(self) }
             }
+
+            /// Produces a vector where every lane has the square root value
+            /// of the equivalently-indexed lane in `self`
+            #[inline]
+            pub fn sqrt(self) -> Self {
```

Comment:
I think this should possible go in another file instead of the main float.rs file.  Also, it needs to be gated on the std cargo feature.

Reply:
Got it - does `vector/fops.rs` sound good?

Reply:
Hm, I browsed the files and maybe this file is the right place to put it (for now at least).  But it should still be gated on the std feature.

Reply:
So will we make sqrt available in core later on? or is there some sort of plan there.

Reply:
Yeah, I think it will using the vector libm (#109), but will require LLVM changes too.